.github/workflows/scripts/sensitive_file_check.py

```
@@ -0,0 +1,123 @@
+    for directory in directories:
+        for root, _, filenames in os.walk(directory):
+            for filename in filenames:
+                full_path = os.path.join(root, filename)
+                # Normalize path separators to forward slashes for consistency
+                full_path = full_path.replace(os.path.sep, "/")
+                for pattern in compiled_patterns:
+                    if pattern.search(full_path):
+                        sensitive_files.append(full_path)
+                        break
+
+    return sorted(list(set(sensitive_files)))
```

Ruff (0.14.11)

Unnecessary `list()` call within `sorted()`

Remove the inner `list()` call

(C414)